frontend/public/components/default-resource.tsx

```diff
@@ -163,21 +253,26 @@ export const DefaultList: React.FC<TableProps & { kinds: string[] }> = (props) =
         transforms: [sortable],
         props: { className: tableColumnClasses[1] },
       },
-      {
-        title: t('public~Created'),
-        sortField: 'metadata.creationTimestamp',
-        transforms: [sortable],
-        props: { className: tableColumnClasses[2] },
-      },
+      ...additionalPrinterColumnsHeaders,
       {
         title: '',
         props: { className: tableColumnClasses[3] },
       },
     ];
+
+    if (!checkColumnsForCreationTimestamp(additionalPrinterColumns)) {
+      headers.splice(2, 0, {
```

Comment:
Debating if created should always come last, which would be more consistent with our other tables